lib/shopify/utils.ts

```diff
@@ -0,0 +1,121 @@
+import Client from 'shopify-buy';
+
+/** Possible donation amounts in dollars */
+export type DonationAmount = 0 | 5 | 10 | 25 | 50 | 75 | 100;
```

Comment:
Are we expecting the set of donation amounts to possibly change in the future? If yes, i believe rcd would have to also change this type along with `env.SHOPIFY_PERMIT_PRODUCT_ID` which isn't super obvious.

I think we should either make a note to document this somewhere or make the type more generic.

Reply:
Good point. I'll add an extra note that the environment variable also needs to be changed.